app/styles/components/explore-side-bar.scss

```diff
@@ -0,0 +1,16 @@
+.g-map{
```

Comment:
Please scope this inside a class. We don't want this to affect all g-map instances.